### ISSUE TITLE
什么时候默认支持https

### ISSUE THREAD

**Issue by lijie815917** [Bilibili/ijkplayer#2516]
什么时候gradle依赖包加入https

**Comment by blueseaguo** [comment]
直接加上配置协议，就支持https

**Comment by hotbiet** [comment]
Can someone please compile the libraries with https. I can not do it. My Email Adress is toslo@live.de
Thanks and Best Regards from Germany

**Comment by lijie815917** [comment]
不支持的，默认，需要编译ffmpeg @blueseaguo

**Comment by hotbiet** [comment]
Yes and exactly that I do not go. I tried with grandle but I get no .so files. Have now installed ndk and I still do not get it.
Can someone please help me and send?

**Comment by CarGuo** [comment]
https://github.com/CarGuo/GSYVideoPlayer  个人编译的项目，里面的so已经支持https，可以试试。

**Comment by hotbiet** [comment]
Thank you CarGuo,

Have ffmpeg re-compiled yesterday.. Channels with https go now. I do not have a sound anymore. I will look at your video player today and learn before.
What can it lie that I have no more sound when I re-compile ffmpeg. If I take the old libary is the sound again there but no more https.

Best Regards

**Comment by hotbiet** [comment]
Danke... funktioniert nun alles. Hab nun ein neues Problem. Wie kann mannicht die url alle 5 Sekunden neu laden ?
